src/components/endowed-positions/EndowedPositionsSearch.js

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useState } from 'react';
+import { navigate } from 'gatsby';
+import SbEditable from 'storyblok-react';
+
+const EndowedPositionsSearch = (props) => {
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const handleInputChange = useCallback((event) => {
+    setSearchTerm(event.currentTarget.value);
+  }, []);
+  const handleSearch = useCallback(() => {
+    navigate(`/endowed-positions/search?term=${searchTerm}`);
+  }, [searchTerm]);
+  const handleKeyDown = useCallback((event) => {
+    if (event.key === 'Enter') {
+      handleSearch();
+    }
+  });
+
+  return (
+    <SbEditable content={props.blok}>
+      <input
+        // className="endowed-positions__input"
+        // id="search-input"
+        onChange={handleInputChange}
+        onKeyDown={handleKeyDown}
+        value={searchTerm}
+      />
+      <button type="button" onClick={handleSearch}>
+        go
+      </button>
+    </SbEditable>
```

Comment:
If this component isn't editable in Storyblok, you might even be able to take out the SBEditable wrapper and the `props` in the declaration line, but I'm going to go ahead and approve this since you're just testing out if you can add a component this way via Storyblok @wmui51